pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
+    "astunparse",
+    "numpy",
+    "ninja",
+    "pyyaml",
+    "setuptools",
+    "cmake",
+    "cffi",
+    "typing_extensions",
+    "future",
+    "six",
+    "requests",
+    "dataclasses",
+]
+build-backend = "setuptools.build_meta:__legacy__"
+
+
+[tool.black]
+# Uncomment if pyproject.toml worked fine to ensure consistency with flake8
+# line-length = 120
+skip-string-normalization = true
+target-version = ["py37", "py38", "py39"]
+include = '''
```

Comment:
I'd prefer if we didn't duplicate the file listings here and `.lintrunner.toml`, and kept a single source of truth for what various formatters/linters run on.

That said, this list isn't the worst offender since it only affects black when called recursively.